codeql-azure-pipeline.yml

```diff
@@ -17,18 +17,18 @@ steps:
 
 - script: |
     npm install
-    npm test
-  displayName: 'npm install and test'
+  workingDirectory: fujiji-server/
```

Comment:
Ah okay, I thought that was the case because of this lol. Lgtm, then!

Reply:
Makes sense, I wonder whether I'm supposed to do npm install for the fujiji-client or not, or even npm install is needed, looking into this!

Reply:
Turns out we don't even need to do that, just need to do init and analyze in CodeQL for JavaScript